k8t/filters.py

```diff
@@ -82,19 +82,17 @@ def hashf(value, method="sha256"):
 
 
 def get_secret(key: str, length: int = None) -> str:
+    provider_name = config.CONFIG.get("secrets", {}).get("provider")
+    if not provider_name:
+        raise RuntimeError("Secrets provider not configured.")
```

Comment:
could just go with an exception here

```
provider_name = None

try:
  provider_name = config.CONFIG['secrets']['provider'].lower()
except KeyError:
  raise RuntimeError("Secrets provider not configured.")
```

Reply:
Yeah, it's just kinda artifact from previous implementation.